index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="">
+      <ul>
+        <li><input type="text" placeholder="Full Name"></li><hr>
+        <li><input type="email" placeholder="Email address"></li><hr>
+        <li><textarea name="Comment" placeholder="Enter text here" rows="5"></textarea></li>
```

Comment:
:red_circle: Kindly make sure you add HTML5 validation rules to prevent malformed data from being sent to the server. 
- [ ] All your inputs should have a required attribute to make sure they are filled out before submitting the form.
- [ ] The Full Name field should accept no more than 30 characters `(maxlength="30")`
- [ ] The Text Field should accept no more than 500 characters `(maxlength="500")`

Reply:
Thank you for the review, I will implement your suggestions